packages/schematics/src/command-line/affected-apps.ts

```diff
@@ -143,6 +143,24 @@ export function affectedE2eNames(
     .map(p => p.name);
 }
 
+export function affectedLibNames(
```

Comment:
This function is great. You can also write a similar function called `affectedBuildableNames` and 
```ts
.filter(p => p.architect.build)
```